charts/timescaledb/values.yaml

```diff
@@ -335,10 +353,13 @@ master:
   priorityClassName: ""
   ## Additional PostgreSQL Master Volume mounts
   ##
-  extraVolumeMounts: []
+  extraVolumeMounts: 
+    - name: postgresql-config
+      mountPath: /bitnami/postgresql/data/postgresql.conf
```

Comment:
I'm not sure about the mount path but I'll test it

Reply:
Thanks 🙂